### PR TITLE
refactor: clean up NST exits related code

### LINKED ISSUE #197: include exitsTokenData in Exit struct

### PR DESCRIPTION
**Pull request by troggy**
- drop ExitStartedV2 event in favour of ExitStarted
- put token data into Exit struct

Resolves #197, #198

### CODE REVIEW

File: contracts/ExitHandler.sol [hunk 1]
```diff
@@ -22,30 +22,29 @@ contract ExitHandler is IExitHandler, DepositHandler {
...
+  /**
+    - tokenData — (optional) NST data
+   */
   struct Exit {
     uint256 amount;
     uint16 color;
     address owner;
     bool finalized;
     uint32 priorityTimestamp;
     uint256 stake;
```
Comment (johannbarbie) [inline contracts/ExitHandler.sol]: if you make stake a uint32 (for example multiple of 0.1 ETH), then you save one storage slot.

**Comment by johannbarbie** [comment]
also this comes to mind: https://github.com/leapdao/leap-contracts/issues/208

File: contracts/ExitHandler.sol (hunk 1 above)
Comment (johannbarbie) [reply contracts/ExitHandler.sol]: out of scope, but noted here: https://github.com/leapdao/leap-contracts/issues/207

**Comment by johannbarbie** [comment]
is there a pr in node that reflects the changes in ABI?

**Comment by troggy** [comment]
@johannbarbie https://github.com/leapdao/leap-node/pull/281

includes the changes for leapdao/leap-node#131 as well (drops NewHeight event)

**Comment by troggy** [comment]
testing this right now as you suggested in https://github.com/leapdao/leap-node/pull/131

**Comment by troggy** [comment]
`Submission` event signature is different for updated contracts, so the updated node won't catch Submission even from _old_ contracts (before upgrade). Will figure out consequences and how to solve

**Comment by troggy** [comment]
NewHeight even is not used in leap-node, so not a problem

**Pull request merged**